package.json

```diff
@@ -24,6 +24,7 @@
     "lib": "lib"
   },
   "scripts": {
+    "start": "tsc src/dataViewBuilder/matrixBuilder.ts && node src/dataViewBuilder/matrixBuilder.js",
```

Comment:
Do not change start command.
To compile matrixBuilder.ts import everything from this file in index.ts